getsmarter_api_clients/geag.py

```diff
@@ -265,4 +269,3 @@ def create_enterprise_allocation(
             )
             logger.error(message)
             raise
-        return response
```

Comment:
[clarification/curious] Why no longer return the `response`?

Reply:
Seems like a bad rebase or something, we just wrote a PR yesterday to add the line to return the response. :)

Reply:
Yep, good catch. Was moving quickly at the end of the day...